### ISSUE TITLE
Move current integration tests to commons module

### ISSUE THREAD

**Issue by spyrkob** [wildfly-extras/prospero#84]
Possibly create new integration testsuite as well to test executing from CLI and integrating with WFLY

**Comment by TomasHofman** [comment]
Working on this.

**Comment by TomasHofman** [comment]
What about moving all integration tests into a new module, where they could live in separate packages (for CLI, basic API, ...)? We could reuse config files, utilities, etc.

**Comment by spyrkob** [comment]
Yes, the new testsuite should be a separate module. Initially I think only tests from `prospero-cli/src/test/java/integration/` would live there, but we'll add more
